auth/ENVOY.sh

```diff
@@ -25,12 +25,6 @@ fi
 
 ### SERVER CONFIGURATION
 
-# These steps are needed to enable local calls to the Cloud Datastore API.
-# This is required when the registry-server is run locally.
-
-# Optionally run this to update your application-default credentials.
-#gcloud auth application-default login
-
 # This assumes that the current gcloud project is the one where data is stored.
```

Comment:
This comment should probably be the same as the one on line 33 of LOCAL.sh

Reply:
Fixed.